### DISCUSSION TITLE
PTU sound

### DISCUSSION THREAD

**Discussion by rpoulachon** [flybywiresim/aircraft#4060] [General]
Hi,
I have installed the latest project in development branch and I didn't heard any PTU sound. Of course I have checked the PTU option in EFB.
Could you explain what I have forgotten to do ?
Thks

Remy

**Comment by 4791pg** [comment]
same here

**Comment by hotshotp** [comment]
You start engine 2 and wait before starting the first one. The ptu logic as of right now is not realistic and will be improved soon.

**Reply by rpoulachon** [reply]
Thanks you very much, but I think I proceed as you mentionned. After starting the engine 2, I have'nt heard any special sound, perharps I have to wait a more Long Time. 

Thanks you.

Best regards
